src/test/scala/uk/gov/hmrc/test/ui/pages/section2/AuthorisationsYesNoPage.scala

```diff
@@ -19,17 +19,29 @@ package uk.gov.hmrc.test.ui.pages.section2
 import uk.gov.hmrc.test.ui.pages.base.Constants.{Clearance, Common, none, yesNo}
 import uk.gov.hmrc.test.ui.pages.base.TariffLinks._
 import uk.gov.hmrc.test.ui.pages.base.{BasePage, Detail}
+import uk.gov.hmrc.test.ui.pages.section1.DeclarationChoicePage.{isClearance, isOccasional}
+import uk.gov.hmrc.test.ui.pages.section1.DetailKeys.AdditionalDeclarationType
 import uk.gov.hmrc.test.ui.pages.section2.DetailKeys.{NoAuthorisationRequired, ProcedureChoice}
 
 object AuthorisationsYesNoPage extends BasePage {
 
-  def backButtonHref: String = ProcedureChoicePage.path
+  def backButtonHref: String = if (isOccasional) OtherPartiesInvolvedListPage.path
+  else if(isClearance) ConsigneeDetailsPage.path
+  else ProcedureChoicePage.path
   val path: String = "/declaration/is-authorisation-required"
```

Comment:
One blank row on top of this row

Reply:
I cannot see a blank row between L30 and L31 yet

Reply:
Done